dev_scripts/find_images_to_rebuild.py

```diff
@@ -50,5 +62,21 @@
     dry_run=True, **kwargs)
 
 handler = RebuildImagesOnRPMAdvisoryChange()
+
+if args.cassete_path:
+    VCRPY_MODE = "none"
+    try:
+        file_dict = yaml.safe_load(args.cassete_path)
```

Comment:
You shouldn't need to load the cassette yourself. As long as you set the Freshmaker `vcrpy_path` configuration to the directory of the cassette and the cassette is named after the Freshmaker event ID, then this will just work.